gulpfile.js

```diff
@@ -1,78 +1,48 @@
 "use strict";
 
+const NO_MODULES = '!**/node_modules/**';
```

Comment:
This is meant to exclude anything found in node_modules folders so the NO is meant to meant "don't look in modules folders", it's not meant as an abbreviation of NODE.

---
In reply to: [87662095](https://github.com/openT2T/translators/pull/119#discussion_r87662095) [](ancestors = 87662095)